index.html

```diff
@@ -388,9 +388,101 @@ <h2 id="protocol-registry">
     <h3>
       Inclusion criteria
     </h3>
+    <aside class="note">
+      The below criteria are a work in progress and are likely to change as
+      this document evolves.
+    </aside>
+    <p>
+      To be included in the registry, the [=digital credential/exchange
+      protocol=]:
+    </p>
+    <ol>
+      <li>MUST be standardized at a recognized standards organization and have
```

Comment:
```suggestion
      <li>MUST be standardized at a recognized standards development organization (SDO) and have
```

Reply:
Also, does W3C maintain a list that we can reference?

Reply:
Possibly https://www.w3.org/liaisons/

Reply:
Actually, yes... that's the right one.

Reply:
Updated the suggestion... dropped the abbreviation, as we don't repeat it.